scipy/sparse/linalg/tests/test_propack.py

```diff
@@ -111,10 +111,13 @@ def test_svdp(ctor, precision, irl, which):
 @pytest.mark.parametrize('precision', _dtype_testing)
 @pytest.mark.parametrize('irl', (False, True))
 def test_examples(precision, irl):
```

Comment:
It might be similarly beneficial (as for `test_basic`) to rename this to something a bit more self-explanatory?

Reply:
Yeah perhaps, but it's easy to find still so not really bothered. The problem with `test_basic` specifically is that it's annoying to grep for since we have a ton of tests named exactly that.